### PR TITLE
Metrics moved to new template

### PR DESCRIPTION
**Pull request by ninlar**
Use new template for API doc for Metrics
This API doc must be curated manually due to the unique structure of the API.
This PR contains Andrew's changes to bring it inline with the new API documentation.

### CODE REVIEW

**Review by GiselleLeeOSI — CHANGES_REQUESTED** [review]
Looks good with simple suggested fixes

**Review by digitaljoshua — APPROVED** [review]
I agree with Ji Won's feedback. I'll approve so you can merge after resolving her comments.

File: content/api-reference/operations/operations-metrics.md [hunk 1]
```diff
@@ -6,94 +6,52 @@ uid: operationsMetrics
...
 
-Id of the tenant for this metric.
+### Parameters
 
-```csharp
-[Required]
-string namespaceId
-```
+`string tenantId`
+Id of the tenant for this metric.
```
[the remark this answers is not in the document]
Comment (ninlar) [reply content/api-reference/operations/operations-metrics.md]: Resolved

File: content/api-reference/operations/operations-metrics.md [hunk 2]
```diff
@@ -135,63 +82,43 @@ Retrieves metrics related to OMF ingress rates for a given namespace.
...
```
[the remark this answers is not in the document]
Comment (ninlar) [reply content/api-reference/operations/operations-metrics.md]: Resolved

File: content/api-reference/operations/operations-metrics.md [hunk 3]
```diff
@@ -135,63 +82,43 @@ Retrieves metrics related to OMF ingress rates for a given namespace.
...
 
+`string namespaceId`
 Id of the namespace for this metric.
 
-```csharp
-[Required]
-string metricId
-```
+`string metricId`
+Id of metric
```
[the remark this answers is not in the document]
Comment (ninlar) [reply content/api-reference/operations/operations-metrics.md]: Resolved

File: content/api-reference/operations/operations-metrics.md [hunk 4]
```diff
@@ -135,63 +82,43 @@ Retrieves metrics related to OMF ingress rates for a given namespace.
...
 
-```csharp
-[Required]
-string namespaceId
-```
+`string tenantId`
+Id of the tenant for this metric.
 
+`string namespaceId`
 Id of the namespace for this metric.
```
[the remark this answers is not in the document]
Comment (ninlar) [reply content/api-reference/operations/operations-metrics.md]: Resolved

**Review by GiselleLeeOSI — APPROVED** [review]
Looks good. Let me know if you want this merged.

**Comment by GiselleLeeOSI** [comment]
Thomas Porter provided the updated file. Merged that one and will use it instead.

**Pull request closed without merging**